homework/week3/2025320058-twuel0921/No12919.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+//A와 B 2 풀이 코드
+public class No12919 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String s = sc.next();
+        String t = sc.next();
+        System.out.println(compareStrings(s, t) ? 1 : 0);
+    }
+    private static boolean compareStrings(String s, String t) {
+        if(s.length() == t.length()) {
+            return s.equals(t);
+        }
+        boolean a = false;
```

Comment:
스타일 차이이기도 한데, 아래처럼도 가능할겁니다
```suggestion
        boolean a = t.endsWith("A") && compareStrings(s, t.substring(0, t.length()-1));
```